src/scope/explainer.11tydata.yaml

```diff
@@ -4,6 +4,8 @@ created: 2020-12-15
 changes:
   - time: 2021-08-24
     log: Link to syntax comparison
+  - time: 2022-10-03
+    log: Document scopes without selectors
```

Comment:
@mirisuzanne It's not clear to me how to add a credit link, e.g. to my Twitter or Github profile, or whether that's even appropriate here. Would you prefer just a parenthetical "(Thanks to Dan Fabulich)" here?

Reply:
That's fine. Markdown syntax is also ok to use in the log. (and multiline logs are ok, Just start them the `|` and a line break: 

```yaml
changes:
  - time: 2021-08-24
    log: Link to syntax comparison
  - time: 2022-10-03
    log: |
      Mulitline log
      with [markdown](#etc)
```